add interface: userid,password and webhdfsUrl. fix: Bind error of

yuichi-tanaka opened the pull request:
fix: Bind error of BigInsights on Cloud
add interface: userid,password and webhdfsUrl

BigInsights on Cloud Basic not supported the Bluemix I/F.
please merge this repository.

sathipal commented:
@yuichi-tanaka Thank you for the pull request. Can you please elaborate the changes? 
We will verify the changes and merge.

yuichi-tanaka commented:
Hi, sathipal
Thank you for replay.

I was going to use it, but it was broken.
BigInsights on Cloud Basic was not supported the Bind of Bluemix now.
I removed the credential check function and added webhdfsUrl interface.
